### ISSUE TITLE
Only emit blob, no download

### ISSUE THREAD

**Issue by bakkertjebrood** [kempsteven/vue-html2pdf#5]
Great package! Very well built, my compliments. Is there any way to only emit the blob and prevent the download of the generated pdf?

**Comment by kempsteven** [comment]
Hey man really appreciate that!
Unfortunately there isn't a way yet, but i think i will make that as a new feature.

But if you can there is a hacky way to do that.

Set the prop ```preview-modal``` to true, that will prevent the download but a preview modal will appear, you hide that by setting a  style ```display:none``` to the modal, class called ```.pdf-preview ```

You need to use /deep/ or ::v-deep so you can style the component.
Sample:
```
/deep/ .pdf-preview { display: none; }
```
Not sure yet if this will work, cant test it right now since im in my phone, but if you can wait i can make this feature in a few days or so

**Comment by kempsteven** [comment]
its now available in version 1.3.6 yey, you can now disable the download by setting these props to false:
```
...
:enable-download="false"
:preview-modal="false"
...
```

**Comment by kkarkos** [comment]
Hi @kempsteven , I also require the Blob only. With 
```
...
:enable-download="false"
:preview-modal="false"
...
```
**hasDownloaded** is not being triggered. You can reproduce on the demo page. And you also mention it in the docs. 

```
This will NOT be triggered if the props `enable-download` AND `preview-modal` is set to false.
```

Maybe this has been removed again?   

I started using the hack you mentioned above, which works for now.
